crates/ruff/src/message/github.rs

```diff
@@ -0,0 +1,65 @@
+use crate::fs::relativize_path;
+use crate::message::{Emitter, EmitterContext, Message};
+use crate::registry::AsRule;
+use std::io::Write;
+
+/// Generate error workflow command in GitHub Actions format.
+/// See: [GitHub documentation](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message)
+#[derive(Default)]
+pub struct GithubEmitter;
+
+impl Emitter for GithubEmitter {
```

Comment:
My read on the Rust naming rules is that this is correct (over `GitHubEmitter`).

Reply:
I don't know. I choice this naming to keep it consistent with `SerializationFormat::Github`